### ISSUE TITLE
 FaceLandmarker.createFromOptions works fine on web and android devices on dev tools, fails on ios devices in dev tools inside a webworker

### ISSUE THREAD

**Issue by Noob7** [google/mediapipe#5123]
### Have I written custom code (as opposed to using a stock example script provided in MediaPipe)

None

### OS Platform and Distribution

Mac chrome google chrome responsive view

### Mobile device if the issue happens on mobile device

all simulated iphone devices 

### Browser and version if the issue happens on browser

chrome Version 119.0.6045.159 (Official Build) (arm64)

### Programming Language and version

javascript

### MediaPipe version

0.10.3

### Solution

FaceLandmarker

### Describe the actual behavior

On simulated ios devices we see an error saying document is not defined

### Describe the expected behaviour

Landmarker should be created 

### Standalone code/steps you may have used to try to get what you need

We have the following code to setup the faceLandmarker. It works fine in chrome. However when we go to responsive view and switch to iphone then it fails with the error :

```shell 
Uncaught (in promise) ReferenceError: document is not defined
    at la (tasks-vision@0.10.7:7:61596)
    at Sa.createFromOptions (tasks-vision@0.10.7:7:93494)
    at createFaceLandmarker (worker:21:46)
```

```shell 
async function createFaceLandmarker(FaceLandmarker, FilesetResolver) {
  const filesetResolver = await FilesetResolver.forVisionTasks(
    "https://cdn.jsdelivr.net/npm/@mediapipe/tasks-vision@0.10.3/wasm"
  );
  faceLandmarkerLocal = await FaceLandmarker.createFromOptions(filesetResolver, {
    baseOptions: {
      modelAssetPath:'/models/face_landmarker.task',
      // modelAssetPath: `https://storage.googleapis.com/mediapipe-models/face_landmarker/face_landmarker/float16/1/face_landmarker.task`,
      delegate: "GPU"
    },
    outputFacialTransformationMatrixes: true,
    runningMode:'IMAGE',
    numFaces: 1
  });
}
```

**Comment by Noob7** [comment]
![Uploading Screenshot 2024-02-06 at 2.20.26 PM.png…]()

**Comment by Noob7** [comment]
![Screenshot 2024-02-06 at 2 20 26 PM](https://github.com/google/mediapipe/assets/10553468/7926d5f0-d18c-4625-a2db-a63da635716a)
![Screenshot 2024-02-06 at 2 20 37 PM](https://github.com/google/mediapipe/assets/10553468/fb103629-2da9-43c4-be27-fa68369b6e9c)

**Comment by schmidt-sebastian** [comment]
Will take a look.

**Comment by kuaashish** [comment]
Hi @Noob7,

We are currently investigating this matter. In the meantime, could you please try using the latest available version, which is 0.10.9? It appears that you are currently using an older version 0.10.3.

Thank you!!!
